preparation.py

```diff
@@ -60,3 +77,12 @@ def preprocess_image(img, target_size=(128, 128)):
     else:
         x = torch.autograd.Variable(x, volatile=True)
     return x
+
+
+def add_dropped_objects(original, processed):
+    reconstructed = processed
+    labeled = label(original)
+    for i in range(1, labeled.max() + 1):
+        if np.sum((labeled == i) * processed) == 0:
```

Comment:
@apyskir we are checking whether there i a pixel that is both labeled and was 0 in the processed right? So maybe np.any(np.where(~(labeled==i) & processed)) would be more readable.

Reply:
@jakubczakon No, we are checking if all pixels labeled as `i` in `labeled` image are 0 in `processed` image. I will find a more readable solution.

Reply:
@jakubczakon I can see that your solution works, so I include it.